### PR TITLE
Introduce `TkAlJetHT`

### PR DESCRIPTION
**Pull request by mmusich**
#### PR description:

In PR https://github.com/cms-sw/cmssw/pull/27307 the `TkAlMinBias` ALCARECO flavour was added to the `JetHT` dataset in the `autoAlCa` file, in order to have it produced for the Run-2 Ultra-Legacy reprocessing.
Operational experience with it has indicated that the resulting PDs are too large to be able tocomfortably distribute them at the target `T2`s, so that workflow is superseded by adding a new producer `TkAlJetHT` tailored to be used on the `JetHT` sample, which contains a prescale of 10 (1 event of ten input events will be processed). This should alleviate the output size and the burden on disk and operation teams.
This is proposed in this PR. As a new  `eventSetupPathsKey`  (= `TkAlJetHTHLT`) is introduced here a new set of `AlCaRecoTriggerBitsRcd` tags:
   * `AlCaRecoHLTpaths_2017_offline_v6` which is a superset of the production tag `AlCaRecoHLTpaths_2017_offline_v4`, but will all IOVs substituted by new payloads adding the `TkAlJetHT` key (used in `auto:run2_data*`)
   * `AlCaRecoHLTpaths_2021_offline_v2`  which is a superset of the production tag `AlCaRecoHLTpaths_2021_offline_v1`, but will all IOVs substituted by new payloads adding the `TkAlJetHT` key (used in `auto:run3_data*`)

#### PR validation:

Run successfully the Tier-0 testing workflow:
```console
runTheMatrix.py -l 1000.0
```

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Not a backport but it will need to be backported (at least to 12.4.X)

### CODE REVIEW

**Comment by mmusich** [comment]
please test

**Comment by tvami** [comment]
type tracking

File: Configuration/AlCa/python/autoCond.py [hunk 1]
```diff
@@ -24,11 +24,11 @@
     # GlobalTag for MC production (p-Pb collisions) with realistic alignment and calibrations for Run2
     'run2_mc_pa'                   : '123X_mcRun2_pA_v1',
     # GlobalTag for Run2 data reprocessing
-    'run2_data'                    : '123X_dataRun2_v3',
+    'run2_data'                    : '124X_dataRun2_Candidate_2022_06_06_10_25_34',
```
Comment (mmusich) [inline Configuration/AlCa/python/autoCond.py:27]: @cms-sw/alca-l2 please provide suitable GTs

Comment (tvami) [reply Configuration/AlCa/python/autoCond.py:27]: Will do. Do you prefer if we give it to you, or have a separate dedicated PR?

Comment (mmusich) [reply Configuration/AlCa/python/autoCond.py:27]: I would prefer to have it self-consistent.

**Comment by tvami** [comment]
We've created
```
124X_dataRun2_v1
124X_dataRun2_HEfail_v1
124X_dataRun2_relval_v1

124X_dataRun3_v1
124X_dataRun3_relval_v1
```

The diffs are
https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts//123X_dataRun2_v3/124X_dataRun2_v1
https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts//123X_dataRun2_HEfail_v3/124X_dataRun2_HEfail_v1
https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts//123X_dataRun2_relval_v4/124X_dataRun2_relval_v1
https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/123X_dataRun3_v7/124X_dataRun3_v1
https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/123X_dataRun3_relval_v6/124X_dataRun3_relval_v1

**Comment by mmusich** [comment]
@cmsbuild, please test

**Comment by tvami** [comment]
+alca
 * test produce the newly created ALCARECO: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-172411/25312/runTheMatrix-results/1000.0_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT/TkAlJetHT.root

**Comment by emanueleusai** [comment]
+1

**Comment by qliphy** [comment]
+1

**Pull request merged**